drivers/hmis/app/graphql/schema.graphql

```diff
@@ -32,16 +32,28 @@ HUD Assessment
 """
 type Assessment {
   assessmentDate: ISO8601DateTime!
```

Comment:
Not super important but this should probably be a `ISO8601Date` because the input type for it is a `yyyy-mm-dd` string

Reply:
Done